addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -371,6 +372,59 @@ const Wysiwyg = Widget.extend({
     redo: function () {
         this.odooEditor.historyRedo();
     },
+    /**
+     * Start or resume the date field changes muation observers.
+     *
+     * Necessary to keep all copies of a given field at the same value throughout the page.
+     */
+    _observeDateFieldChanges: function () {
+        if(this.dateObservers) {
+            for (let observerData of this.dateObservers) {
+                observerData.observer.observe(
+                    observerData.field,
+                    { characterData:true, subtree: true, childList: true }
+                );
+            }
+        } else {
+            const dateFieldSelector = '*[data-oe-field][data-oe-type=datetime], *[data-oe-field][data-oe-type=date]'
+            const $dateFields = this.$editable.find(dateFieldSelector);
+            this.dateObservers = [];
+
+            $dateFields.each((i, field) => {
```

Comment:
C'est indispensable de tant utiliser jquery ? On peut pas juste itérer en `for...of` natif ? Les `find` ça peut pas être des `querySelector` ? `addClass` être `classList.add` etc etc ? Je trouve ça un peu triste d'écrire du *nouveau* code qui utilise autant jquery pour des features qui existent en natif quoi... ^^' Après y'a p-e une bonne raison que je ne vois pas là tout de suite hein, genre tel ou tel comportement natif qui est différent de jquery et qui ne t'arrange pas ?

Reply:
y a pas une raison particulière. a part que je suis plus a l'aise avec jQuery et que de toute façon y a du jQuery partout dans website. Si tu veu je refactor.